### DISCUSSION TITLE
Pods @post_date doesn't honour WP's date setting (and is also not user-configurable)

### DISCUSSION THREAD

**Discussion by patrikhuber** [pods-framework/pods#6700] [Ideas]
Hi,

In WP's Settings => General, my WP date format is set to `j F Y`. This works fine everywhere on the website.

However when adding a Pod and outputting `{@post_date}`, it shows (for example) `2020-11-27 11:00`.

Users have noted this in 2014 and following - here an old Google Cache link to the old Pods support forum: https://webcache.googleusercontent.com/search?q=cache:v17EuboNpB4J:https://pods.io/forums/topic/date-formatting-using-magic-tags/+&cd=1&hl=en&ct=clnk&gl=uk

This is **not** a duplicate of #5038. #5038 is about custom fields - on these, the date is user-configurable in the Pods custom fields settings.
However the `{@post_date}` fields is not user-configurable.

The workaround in above link is to add a PHP function that converts the date. However adding a PHP function is an obstacle to many. It would be great if:
1) Pods honoured WP's date setting (bugfix)
2) There was a Pods way of setting the date output format without using PHP - ideally this would even be at the time of outputting it so that I could output the date of the same Pods type *with* the time in one Pods template and *without* the time in another Pods template. (feature request)

**Comment by patrikhuber** [comment]
I think the best solution to all this, if technically feasible, would be if we could just access the individual elements of `{@post_date}`'s date. It would be fantastic to be able to do, for example, `{@post_date.month}/{@post_date.year}`. Some people in above forum link also suggested this and it would be a very intuitive way and solve all problems.

**Comment by JoryHogeveen** [comment]
Hi @patrikhuber 

Please see #5676 as a concept for such a feature.

**Comment by patrikhuber** [comment]
Hi @JoryHogeveen,

Cool, that's awesome! I'll leave a comment there. :-)
